truss/util/path.py

```diff
@@ -86,19 +85,6 @@ def build_truss_target_directory(stub: str) -> Path:
     return target_directory_path
 
 
-def calc_shadow_truss_dirname(truss_path: Path) -> str:
```

Comment:
Moved to `truss_gatherer` to prevent a circular dependency, now that `hash.py` depends on `truss/util/path.py`